bin/notify-on-job-fail

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+: "${SLACK_TOKEN:?The SLACK_TOKEN environment variable is required.}"
+: "${SLACK_CHANNELS:?The SLACK_CHANNELS environment variable is required.}"
+
+bin="$(dirname "$0")"
+
+aws_batch_job_id="${AWS_BATCH_JOB_ID:-}"
+github_run_id="${GITHUB_RUN_ID:-}"
+
+echo "Notifying Slack about failed ingest job."
+message=":x: Ingest job failed. "
```

Comment:
Have the emojis gone because they can't be passed through as arguments? I find them useful when scanning the (often numerous) messages to the slack channel. 

Formerly the text was "❌ This pipeline has FAILED 😞" (I realise this wasn't sent to slack, but the phylogenetics pipeline does send an analogous message to slack).

Reply:
Agree, the emoji are useful for spotting 'bad' messages!

Reply:
Ah, the `:x:` will show up in the Slack message as ❌
I can keep the original message if it is clearer to have the emojis directly in the script.

Reply:
Added the emoji's back in [dcce209](https://github.com/nextstrain/ncov-ingest/pull/302/commits/dcce209cb321c18592d63068e531ffd57586f05f)

Reply:
> Ah, the :x: will show up in the Slack message as ❌

🤦  my mistake!